docs/examples/NNIG_Hierarchy_NGG.py

```diff
@@ -0,0 +1,207 @@
+"""
+Conjugate Normal Normal-InverseGamma hierarchy for univariate data.
+
+This class represents a hierarchical model where data are distributed
+according to a normal likelihood, the parameters of which have a
+Normal-InverseGamma centering distribution. That is:
+f(x_i|mu,sig) = N(mu,sig^2)
+   (mu,sig^2) ~ N-IG(mu0, lambda0, alpha0, beta0)
+This hierarchy is conjugate, therefore the following methods have to be
+implemented: is_conjugate, like_lpdf, marg_lpdf, initialize_state,
+initialize_hypers, update_hypers, draw, compute_posterior_hypers,
+update_summary_statistics.
+The state is composed of mean and variance. The state hyperparameters,
+contained in the Hypers object, are (mu0, lambda0, alpha0, beta0), all
+scalar values. In the following implementation we assume a prior on the
+hyperparameters.
+"""
+
+import numpy as np
+import scipy.stats as ss
+
+
+def is_conjugate():
+    """
+    :return: True for conjugate, False for non-conjugate hierarchies
+    """
+    return True
+
+
+def like_lpdf(x, state):
+    """
+    Likelihood log-density
+    :param list x: point in which lpdf is evaluated
+    :param list state: model parameters
```

Comment:
please stick to the pydocs stile used in the other files, e.g. https://github.com/bayesmix-dev/pybmix/blob/master/pybmix/core/chain.py

Reply:
We changed them. We think that in some cases it could be useful to specify the return types in the documentation (like when the user has to return a list of lists). Let us know if the way we did it fits the style:
```
    Return
    ------
    list[list]
        updated summary statistics and cluster data values 
```

Reply:
see here https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_numpy.html#example-numpy
for examples on how to specify the type of the list